jwebunit-commons-tests/src/main/java/net/sourceforge/jwebunit/tests/FramesAndWindowsTest.java

```diff
@@ -139,8 +139,6 @@ public void testCloseWindow() {
     @Test public void testGotoInlineFrame() {
         beginAt("InlineFrame.html");
         assertTextPresent("TopFrame");
-        // Is this how it should work? see also the test below
-        assertTextNotPresent("ContentFrame");
```

Comment:
asText() now includes the content of iframe elements in htmlunit 2.24 see: http://htmlunit.sourceforge.net/changes-report.html